main.go

```diff
@@ -155,6 +155,13 @@ func loadConfigs() (config.Group, error) {
 		}
 	}
 
+	// Command line Arg
+	if flag.NArg() > 0 {
+		confString := strings.Join(flag.Args(), "\n")
+		confBody := ":" + config.Port + "\n" + strings.Replace(confString, ",", "\n", -1)
```

Comment:
Fair point. Yes, I do think commas are easier than newlines (spaces only won't suffice) when typing them in the command line. This feature is intended for simple configurations where you just need something up and running real quickly, not as a whole replacement for the Caddyfile. If you need commas in a Caddyfile without using a real file, perhaps pipe it into Caddy?